client/js/verification-portal/components/day.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+import injectSheet from 'react-jss'
+import { Field } from 'redux-form'
+import { connect } from 'react-redux'
+import { change } from 'redux-form'
+import classNames from 'classnames'
+import Time from './time'
+import _ from 'lodash'
+
+const dayName = [
+	'Sunday',
+	'Monday',
+	'Tuesday',
+	'Wednesday',
+	'Thursday',
+	'Friday',
+	'Saturday',
+]
+const closedAllDayStatus = (dayNum, hours) => {
+	return _.get(hours, `[${dayNum}].closedAllDay`) || false
+}
```

Comment:
Nice use of a function to separate the logic and pulling out the `dayName` array! Keeps it clean 😄 

Would you mind changing the `dayNum` variable to use full words so that it's more clear?